src/main/java/com/yurjinia/common/validator/JwtValidator.java

```diff
@@ -0,0 +1,20 @@
+package com.yurjinia.common.validator;
+
+import org.springframework.stereotype.Component;
+
+import java.util.regex.Pattern;
+
+@Component
```

Comment:
@Component needed for those classes that we want to inject.
As this class contains only static elements, and likely it's an infrastructure code - it must be a @UtilityClass.